pkg/splunk/enterprise/configuration.go

```diff
@@ -38,6 +38,32 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/log"
 )
 
+var defaultLivenessProbe corev1.Probe = corev1.Probe{
+	InitialDelaySeconds: livenessProbeDefaultDelaySec,
+	TimeoutSeconds:      livenessProbeTimeoutSec,
+	PeriodSeconds:       livenessProbePeriodSec,
+	ProbeHandler: corev1.ProbeHandler{
+		Exec: &corev1.ExecAction{
+			Command: []string{
+				"/sbin/checkstate.sh",
+			},
+		},
+	},
+}
+
+var defaultReadinessProbe corev1.Probe = corev1.Probe{
```

Comment:
effective golang , use short assignment statement `defaultReadinessProbe := corev1.Probe{`

Reply:
@vivekr-splunk this is the correct declaration outside of functions. Short declaration is only allowed inside the functions. GoLang will throw an error if I use `:=` here.

Reply:
oh didnt see they are global